src/plugins/security_oss/public/insecure_cluster_service/insecure_cluster_service.tsx

```diff
@@ -84,37 +86,30 @@ export class InsecureClusterService {
     };
   }
 
-  public start({ core }: StartDeps): InsecureClusterServiceStart {
-    const shouldInitialize =
```

Comment:
__note:__ We can keep this check here if you wish, but we have it in the AppState service now and the overhead we'll get here should be negligible in theory (unnecessary Observable subscription in `initializeAlert`)